go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	k8s.io/code-generator v0.23.5
 	k8s.io/klog v1.0.0
 	k8s.io/kube-openapi v0.0.0-20220124234850-424119656bbf
-	knative.dev/pkg v0.0.0-20220329144915-0a1ec2e0d46c
+	knative.dev/pkg v0.0.0-20220524202603-19adf798efb8
```

Comment:
Should update this to @main again, and fix changeset.Get calls accordingly. This will require us to build with Go 1.18+

Reply:
Sorry you're right, good point. We should wait for the next knative release that includes the changeset.Get changes, and bump to use that.

Reply:
This could also be two changes: one to bump to 1.5, and a future one (which I can do if you want) to bump to 1.6 and include changeset.Get goodness.

Reply:
Yep, I am fine with both, depends on when this one gets merged 😝

Reply:
FWIW knative/pkg release-1.6 is out: https://github.com/knative/pkg/tree/release-1.6

1.7 should include the changeset changes.

Reply:
Yay I will update to 1.6 then 😇